src/OrchardCore/OrchardCore.Infrastructure.Abstractions/Security/BuiltInRole.cs

```diff
@@ -0,0 +1,22 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace OrchardCore.Security
+{
+    public static class BuiltInRole
```

Comment:
@hishamco How about changing the name to `SecurityConstants` instead? 
Also, in #11920 you said you were going to provide ways to localize these constants, I don't see how localization is will be supported here.

Reply:
`BuiltInRole` is much cleaner and inspired by .NET enum

> Also, in https://github.com/OrchardCMS/OrchardCore/issues/11920 you said you were going to provide ways to localize these constants, I don't see how localization is will be supported here.

It's still WIP don't expect all the things in the first commits ;)